src/syscheckd/src/db/src/dbFileItem.hpp

```diff
@@ -44,13 +44,14 @@ struct FimFileDataDeleter
 class FileItem final : public DBItem
 {
     public:
-        FileItem(const fim_entry* const fim)
+        FileItem(const fim_entry* const fim, bool oldData=false)
```

Comment:
m_oldData could be moved to dbitem.hpp and initialize inside each class because this is a repeated variable